xpp3_min/src/main/resources/META-INF/services/org.xmlpull.v1.XmlPullParserFactory

```diff
@@ -1 +0,0 @@
-org.xmlpull.mxp1.MXParser,org.xmlpull.mxp1_serializer.MXSerializer
```

Comment:
Can we remove this without breaking the fallback?

Reply:
Yeah, this is likely not 100% backwards compatible. If people end up with xmlpull 1.1.x + xpp3_min 1.2.0.Final+, then the xmlpull will fail to load the services. That being said, I am afraid we can not leave the file be as that then break the ServiceLoader code.

I'll do more testing with different combinations and will report back.

Reply:
We can not leave the file as is as it won't work on Java 9 with module path - what happens then is that the ServiceLoader calls fail as the file is invalid and the fallback fails as well, because the xmlpull module is not allowed to access the META-INF/services/* files in different modules.

This also means that the xmlpull + any other old implementation won't work with Java 9 module path. At this point I don't know if there is away to make all these use cases work.